libraries/DataFlash/DataFlash.h

```diff
@@ -286,4 +285,6 @@ class DataFlash_Class
     void dump_structures(const struct LogStructure *structures, const uint8_t num_types);
 
     void Log_Write_EKF_Timing(const char *name, uint64_t time_us, const struct ekf_timing &timing);
+
+    bool NeedPrep();
```

Comment:
If this is private what is it needed for? I didn't see it being used anywhere in DataFlash_Class.